agent/app/agent_capability_unspecified.go

```diff
@@ -26,6 +26,18 @@ import (
 	"github.com/cihub/seelog"
 )
 
+const (
+	capabilityDepsRootDir = "/managed-agents"
+)
+
+var (
+	capabilityExecRequiredBinaries = []string{
```

Comment:
`capabilityExecRequiredBinaries` should be an empty array for unsupported platform.
`capabilityDepsRootDir` should be an empty string for unsupported platform.